### PR TITLE
Add dragonwell on aarch64

### PR DESCRIPTION
**Pull request by joeyleeeeeee97**
This requires https://github.com/AdoptOpenJDK/openjdk-infrastructure/pull/1706
Related issue: https://github.com/AdoptOpenJDK/openjdk-build/issues/2367

### CODE REVIEW

**Review by gdams — APPROVED** [review]

File: pipelines/jobs/configurations/jdk11u_pipeline_config.groovy [hunk 1]
```diff
@@ -121,7 +121,12 @@ class Config11 {
                 arch                : 'aarch64',
                 dockerImage         : 'adoptopenjdk/centos7_build_image',
                 test                : 'default',
-                configureArgs       : '--enable-dtrace=auto'
+                configureArgs       : [
+                        "hotspot" : '--enable-dtrace=auto',
+                        "openj9" : '--enable-dtrace=auto',
+                        "corretto" : '--enable-dtrace=auto',
+                        "dragonwell" : "--enable-dtrace=auto --with-extra-cflags=\"-march=armv8.2-a+crypto\" --with-extra-cxxflags=\"-march=armv8.2-a+crypto\"",
```
Comment (karianna) [inline pipelines/jobs/configurations/jdk11u_pipeline_config.groovy:128]: remove trailing ','

**Review by karianna — CHANGES_REQUESTED** [review]

**Review by M-Davies — APPROVED** [review]

**Comment by joeyleeeeeee97** [comment]
@karianna  Hi, please have another look at this patch? Thanks.

File: build-farm/platform-specific-configurations/linux.sh [hunk 2]
```diff
@@ -134,7 +134,12 @@ if [ "$JAVA_FEATURE_VERSION" -gt 11 ]; then
     "$JDK_BOOT_DIR/bin/java" -version 2>&1 | sed 's/^/BOOT JDK: /'
 fi
 
-if [ -r /usr/local/gcc/bin/gcc-7.5 ]; then
+
+if [ "${VARIANT}" == "${BUILD_VARIANT_DRAGONWELL}" ] && [ "$JAVA_FEATURE_VERSION" -eq 11 ] && [ -r /usr/local/gcc9/ ]; then
+  export PATH=/usr/local/gcc9/bin:$PATH
+  [ -r  /usr/local/gcc9/ ] && export CC=/usr/local/gcc9/bin/gcc-9.3
```
Comment (aahlenst) [inline build-farm/platform-specific-configurations/linux.sh:140]: You already did `[ -r /usr/local/gcc9 ]`, so all those additional checks seem unnecessary. Or do you want to test for the existence of the individual binaries gcc/g++? If the latter, wouldn't a single condition not make more sense? Having gcc 7 and g++ 9 simultaneously enabled does not seem desirable.

File: pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+ARG image
+
+FROM $image
+
+RUN \
+    # Dragonewell 8 requires a dragonwell 8 BootJDK
+    mkdir -p /opt/dragonwell; \
+    wget https://github.com/alibaba/dragonwell8/releases/download/dragonwell-8.5.5_jdk8u275-b2/Alibaba_Dragonwell_8.5.5-FP1_Linux_aarch64.tar.gz; \
```
Comment (aahlenst) [inline pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile:8]: Checksum verification please.

File: pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+    tar -xf Alibaba_Dragonwell_8.5.5-FP1_Linux_aarch64.tar.gz -C /opt/; \
```
Comment (aahlenst) [inline pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile:9]: With `--strip-components=1`, you should be able to extract into the target directory.

File: pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile (hunk 3 above)
Comment (joeyleeeeeee97) [reply pipelines/build/dockerFiles/dragonwell_aarch64.dockerfile:8]: ok

File: build-farm/platform-specific-configurations/linux.sh (hunk 2 above)
Comment (joeyleeeeeee97) [reply build-farm/platform-specific-configurations/linux.sh]: removed this check

**Review by aahlenst — APPROVED** [review]

**Review by karianna — APPROVED** [review]
LGTM

**Pull request merged**